src/main/index.ts

```diff
@@ -75,6 +73,12 @@ async function main() {
 
   // create window manager and open app
   windowManager = new WindowManager(proxyPort);
+
+  /**
+   * This depends on:
+   * 1. windowManager: it will send IPC to the main window for notifications
+   */
+  startUpdateChecking();
```

Comment:
Maybe better to `startUpdateChecking(windowManager)` for explicit dependency and no need in comments? :)

Reply:
Agreed, though I initially had it as a comment as it was more of a transitive dependency (my function used `broadcastIpc`).